### ISSUE TITLE
bug: Floating Action Button for import of new service not reachable with TalkBack when no services in list

### ISSUE THREAD

**Issue by janreppien** [twofas/2fas-android#221]
### Bug type

Other

### App version

5.5.3

### Device environment

POCO X7 Pro

### Bug description

When using the app without having any services the button for adding new services has text besides the +. In this state the Button is not reachable with TalkBack.

How to reproduce:
1. Activate TalkBack
2. Open fresh install of 2FAS
3. Swipe left (or right) until the green focus reached the starting point
4. Notices how TalkBack never focuses on the floating action button

### Additional context

Somewhat related to #148 

### Acknowledgements

- [x] This issue is not a duplicate of an existing bug report.
- [x] I understand that security vulnerabilities should be reported to security@2fas.com instead of on GitHub.
- [x] I have chosen an appropriate title.
- [x] All requested information has been provided properly.
